lib/verifiablePresentation/SSI.types.ts

```diff
@@ -10,7 +10,6 @@ export interface Proof {
   created: string;
   proofPurpose: string;
   verificationMethod: string;
-  jws: string;
```

Comment:
Isn't the jws optional?

Reply:
It should be, but it isn't. I tried to use jws?: string but it's not working.

Reply:
I tried to make it optional but it didn't work. The message is TS2411: Property 'jws' of type 'string | undefined' is not assignable to 'string' index type 'string'. Not sure how to fix it, I'm trying to find it out.

Reply:
I've just had to change [x: string]: string to [x: string]: string | undefined; to make it work.

Reply:
I guess for now it works